Source/WebKit/NetworkProcess/cocoa/NetworkProcessCocoa.mm

```diff
@@ -251,4 +256,38 @@ void saveCookies(NSHTTPCookieStorage *cookieStorage, CompletionHandler<void()>&&
 }
 #endif // HAVE(NW_PROXY_CONFIG)
 
+#if USE(RUNNINGBOARD) && USE(EXTENSIONKIT)
+bool NetworkProcess::aqcuireLockedFileGrant()
+{
+#if __has_include(<WebKit-Swift.h>)
```

Comment:
Ditto.

Reply:
This was resolved by your request to invert the logic above.